e2e/bazel-bzlmod-lock-file-from-args/repo.yaml

```
@@ -0,0 +1,5 @@
+repositories:
+  - baseurl: https://kojipkgs.fedoraproject.org/packages
+    name: fedora
+    priority: 1
+    arch: x86_64
```

```suggestion
repositories:
- arch: x86_64
  metalink: https://mirrors.fedoraproject.org/metalink?repo=fedora-40&arch=x86_64
  name: 40-x86_64-primary-repo
- arch: x86_64
  metalink: https://mirrors.fedoraproject.org/metalink?repo=updates-released-f40&arch=x86_64
  name: 40-x86_64-update-repo
```
The old mirror doesn´t work